sky/execution.py

```diff
@@ -370,6 +383,7 @@ def _execute(
             subprocess_utils.run('sky status --no-show-spot-jobs', env=env)
         print()
         print('\x1b[?25h', end='')  # Show cursor.
+    return job_id, handle
```

Comment:
Seems we don't add a return to `launch` and `exec` functions? Do we need to pump it?

Also, it would be good to add a test in the smoke tests : )

Reply:
Added 🫡